### DISCUSSION TITLE
Plugin Torpedo Wall of Sound has failed while scan

### DISCUSSION THREAD

**Discussion by nnlkcncff** [apohl79/audiogridder#830] [Help]
Torpedo Wall of Sound version 4.4.3 (if you want to test it yourself it is free, but requires registration).

The Torpedo Wall of Sound plugin version of AudioUnit doesn't pass some check during scanning and shows up as "Failed" under section _AudioGridder Server ▸ Plugins ▸ Description_.

First, the base log file name of this plugin does not contain some string (apparently, an abbreviated plugin name or something like that), for comparison, the log file name of another plugin is added:
 - `AudioUnit-Effects-aufx,,TwoN-AudioUnit_2021-11-27_22-47-36.log`
 - `AudioUnit-Effects-aufx,U532,Mcrl-AudioUnit_2021-11-27_23-23-50.log`

I think there must be something between `*aufx,` and `,TwoN*`!

Well, and, in fact, the contents of the journal:

> [22:47:36.397|app|7fced5704278] commandline: -scan AudioUnit:Effects/aufx,,TwoN|AudioUnit
> [22:47:36.399|app|7fced5704278] scan mode: format=AudioUnit id=AudioUnit:Effects/aufx,,TwoN
> [22:47:36.399|server|10c9dfaf0] scanning id=AudioUnit:Effects/aufx,,TwoN fmt=AudioUnit
> [22:47:36.418|app|7fced5704278] ...failed
> [22:47:36.418|app|7fced5704278] initialise complete
> [22:47:36.418|app|7fced5704278] shutdown
> 

Also, the files of this plugin had a different group in the UNIX access rights (compared to other plugins), I made them identical, but it did not help to solve the problem, except VST versions successfully worked (plugin version VST3 was not detected until I disabled the option _Plugin Menu ▸ Disable Server Filter_, apparently, something is wrong with categories), so this is not the case.

The plugin obviously has some problems with the metadata, but is this the case or is AudioGridder not handling this situation correctly?

How to check the plugin itself, or can you check it yourself?
If there is something wrong with it, I would like to provide feedback to the developers.

This type of plugin is hardly worth running remotely, as it cannot be called highly resource-hungry, I'm more worried about AudioGridder scanning inappropriately.

**Comment by apohl79** [comment]
You can try it in reaper, which supports all 3 formats as well.

**Reply by nnlkcncff** [reply]
Although I was initially sure of a successful result, I still tried to connect the plugin in Reaper — successfully, but I also have in Logic Pro X plugin Torpedo Wall of Sound AU, VST, VST3 versions successfully connected.
What's the point of changing a DAW when it doesn't need to be run on the remote host that AudioGridder Server is supposed to run on?

It seems to me, that in this case the problem is on the side of AudioGridder Server, which can't connect plug-in, not on the side of DAW — this is the next step.

Maybe I misunderstood you?

Again, I have no particular need to run this plugin through AudioGridder, but I am concerned that AudioGridder Server is not scanning it correctly, which makes it impossible to provide the plugin to the DAW on the remote host.
Any other plugin could be in place of this one.

How can I help you diagnose and fix this problem?

![image](https://user-images.githubusercontent.com/82151266/143765096-09d2f8d1-f47d-4b3b-bf60-3fd96e588405.png)

**Reply by apohl79** [reply]
yes i meant you should try reaper on the server side and see if the plugin works in reaper. but you said, that at least one plugin format is working. so why not using the working one?

**Reply by nnlkcncff** [reply]
I'm just worried that AudioGridder failed to scan this plugin and am trying to figure out if the problem is with the plugin or with AudioGridder's scanning algorithm.
If it's the scanning algorithm, you could fix it, if it's the plugin, I could inform the plugin developer of the error. I can't do that without you, since I don't understand what the problem is exactly.
